{{cookiecutter.project_slug}}/clients/vue3/src/views/Signup.vue

```diff
@@ -1,41 +1,49 @@
 <template>
-  <div class="signup">
-    <form @submit.prevent="attemptUserRegistration()">
+  <div
+    class="signup container w-full sm:max-w-prose content-center mx-auto py-5 bg-gray-100 rounded-lg"
+  >
+    <h1 class="heading--bold">Sign Up.</h1>
+    <form class="flex flex-col items-center" @submit.prevent="attemptUserRegistration()">
       <InputField
         v-model:value="form.firstName.value"
         :errors="form.firstName.errors"
         @blur="form.firstName.validate()"
-        label="First Name:"
-        placeholder="First name"
+        label="First Name"
+        placeholder="Enter first name..."
```

Comment:
**Info:** Placeholder text should specify a short hint that describes the expected value of the input field so figured it'd make more sense to just add a bit more clarification text here (instead of just the label name).

Type. for all fields on the `Signup` view.